src/styles/blocks/about.scss

```diff
@@ -0,0 +1,45 @@
+.about {
+  &__title {
+    margin: 0 0 48px;
+    font-weight: bold;
+    font-size: 36px;
+    line-height: 36px;
+    align-items: center;
+    letter-spacing: -1.5px;
+    color: #000;
```

Comment:
all the colors should be put in variables

Reply:
Correct, corrected)